doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -109,16 +109,18 @@ putcmdlog <text>
 ^^^^^^^^^^^^^^^^^
 putxferlog <text>
 ^^^^^^^^^^^^^^^^^
-  Description: sends text to the bot's logfile, marked as 'file-area' (x)
+
+Description: logs <text> to the logfile and partyline if the 'files' flag (x) is active via the 'logfile' config file setting and the '.console' partyline setting, respectively.
 
   Returns: nothing
 
   Module: core
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-putloglev <level(s)> <channel> <text>
+putloglev <flag(s)> <channel> <text>
```

Comment:
If "level" stays "flag", the helptext at `tclmisc.c` should be changed accordingly.